### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by alexmedinasf**
Created a desktop version for the Portfolio.

Created a new CSS file to style the desktop version of the website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,10 +15,16 @@
 	<nav class="menu-bar">
         <button type="button" class="name"> Alexandre Ferreira </button>
         <button type="button" class="menu"> <img src="./images/menu.png" alt="menu"> </button>
-    </nav>
+			<ul id="nav-links">
+				<li><a href="#feature-project">Portfolio</a></li>
+				<li><a href="#abme">About me</a></li>
+				<li><a href="#contact-form">Contact</a></li>
+				<li><a href="#contact-form"><img src="./images/maildesktop.svg" alt="Contact"></a></li>
+				</ul>
```
Comment (omarsalem7) [inline index.html:23]: - [ ] Kindly remove the hamburger menu from the desktop version by adding `display:none` in the desktop version.
- [ ] Kindly remove the underline from nav-links to match it like the Figma.
- [ ] Also consider adding the appropriate space to right and left to be like the Figma.

File: css/desktop-style.css [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  position: absolute;
+  inset: auto 20px;
+  flex-direction: column;
+}
+
+#feature-project {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 30px;
+}
```
Comment (omarsalem7) [inline css/desktop-style.css:36]: - [ ] Kindly make feature-project align with projects section like the Figma.

- [ ] Alos consider styling languages in the feature project as stated in the following image.

| Your design  | Figma design|
| ------------- | ------------- |
| 2022_08_24_009_Kleki  |image|

File: css/desktop-style.css [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  max-width: 455px;
+  max-height: 256px;
+  margin-left: 135px;
+  margin-top: 130px;
+}
+
+#contact-form > * {
+  flex: 1;
+}
+
```
Comment (omarsalem7) [inline css/desktop-style.css:150]: - [ ] Kindly use the correct background color as Figma. Color code: #FFFFFF

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @alexmedinasf 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/desktop-style.css [hunk 4]
```diff
@@ -0,0 +1,193 @@
+main > * {
+  padding: 0 7%;
+}
+
+#nav-links {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: 895px;
```
Comment (ShahierNashaat) [inline css/desktop-style.css:8]: - [ ] [Optional] Kindly be noticed that to remove the horizontal scroll bar you have to remove the `margin-left: 895px;` and add these CSS lines.
```css
    position: absolute;
    right: 0;
```

**Review by ShahierNashaat — APPROVED** [review]
Hi @alexmedinasf, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**